open-company/open-company-web
# Cmail video

Card: https://trello.com/c/lmVA2Kwa

To test:
- open Cmail
- click video button
- [x] record a video, did it work? Good
- click video button
- focus url field
- [x] did the field expand and the embed video button appear? Good
- [x] add a NOT valid video url, button is disabled? Good
- [x] add a valid video url, button is now enabled? Good
- [x] add an image (regression test)
- [x] add an attachment (regression test)
- [x] check mobile Cmail for regressions
